docs/mp/grpc/server.adoc

```diff
@@ -31,9 +31,11 @@ include::{rootdir}/includes/mp.adoc[]
 - <<Maven Coordinates, Maven Coordinates>>
 - <<API, API>>
 - <<Usage, Usage>>
+- <<Configuration, Configuration>>
+- <<Examples, Examples>>
 
 == Overview
-The gRPC Microprofile APIs are an extension to xref:{rootdir}/mp/introduction.adoc[Helidon MP] to allow building
+The gRPC Microprofile APIs are an extension to xref:../introduction.adoc[Helidon MP] to allow building
```

Comment:
I think we have been avoiding relative cross-references in case we move the referring file to a different directory at some point. The relative reference would break, whereas using the form with the `rootdir` attribute would still work.

Reply:
resolved across all docs